src/main/js/main_html5.js

```diff
@@ -1152,7 +1152,7 @@ require("../../../html5-common/js/utils/environment.js");
       }
 
       var availableAudio = this.getAvailableAudio();
-      if (availableAudio && availableAudio.length) {
+      if (availableAudio && availableAudio.length  > 1) {
```

Comment:
The update looks good, but even though this is a small change I think it's a good candidate for a unit test. This event might cause issues if it's fired when there's only a single audio track, so that's something we should test for.

Reply:
\+